adoc/SAP-S4HA10-setupguide-sle15.adoc

```diff
@@ -724,7 +724,8 @@ For the ERS and ASCS instances, edit the instance profiles
 profile directory _/usr/sap/{mySid}/SYS/profile/_.
 
 Tell the *{sapStartSrv}* service to load the HA script connector library and
-to use the connector *{s4sClConnector3}*.
+to use the connector *{s4sClConnector3}*. On the other hand, please make sure
+the feature _Autostart_ is *not* used.
```

Comment:
```
On the other hand, please make sure the feature
_Autostart_ is *not* used.
```
should be changed to

Additionally, make sure the feature _Autostart_ is *not* used.